wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Pattern, generateDocs } from 'storybook-django/src/react';
+
+import initCollapsibleBreadcrumbs from '../../../../../client/src/includes/breadcrumbs';
+import template from './breadcrumb-next.html';
+
+const { docs, argTypes } = generateDocs(template);
+
+export default {
+  parameters: { docs },
+  argTypes: {
+    ...argTypes,
+  },
+};
+
+const Template = (args) => {
+  document.addEventListener('DOMContentLoaded', () => {
+    initCollapsibleBreadcrumbs();
+  });
```

Comment:
Just a heads up - this will potentially add multiple listeners, anything inside a react component render function can run multiple times.

Probably not a huge problem for storybook right now but I have nonetheless moved this back up to near the top.

I can explain more on our next call but have a read of these docs to understand a bit more about how React rendering works.

https://felixgerschau.com/react-rerender-components/

https://reactjs.org/docs/rendering-elements.html